src/yadism/coefficient_functions/light/partonic_channel.py

```diff
@@ -4,5 +4,6 @@
 class LightBase(pc.PartonicChannel):
     """Light partonic channel base class."""
 
-    def __init__(self, ESF, nf):
+    def __init__(self, ESF, nf, is_fl11=False):
         super().__init__(ESF, nf)
+        self.is_fl11 = is_fl11
```

Comment:
just a minor detail: this name makes me suspicious ... what I would do instead is define a new partonic channel (i.e. this _class is_ fl11), which only contributes at N3LO - so instead of the `if` branching, just make new contribution (with plain return); this business should be really additive and so we should treat it like this

Reply:
This is ofc possible. 
In the current implementation I tried to use `min_order` to silence the previous orders, 
but I can create new partonic channels if this is clearer.

Reply:
I would say a single class (e.g. `GluonFL11`) is better than `min_order` + `if` - at least to me; and you may be able to reuse it this way ...

Reply:
okay fine